--- system-auth	2018-09-12 18:15:25.306000000 +0000
+++ patched-system-auth	2018-09-12 18:08:58.746000000 +0000
@@ -4,6 +4,8 @@
 auth     optional      pam_faildelay.so
 auth     required      pam_env.so
 auth     required      pam_faillock.so preauth silent deny=5 even_deny_root audit unlock_time=900 root_unlock_time=60 fail_interval=900
+auth 	 [success=1 default=bad]     pam_oath.so usersfile=/etc/liboath/users.oath window=1
+auth     [default=die] pam_faillock.so authfail deny=5 even_deny_root audit unlock_time=900 root_unlock_time=60
 auth     sufficient    pam_sss.so forward_pass
 auth     sufficient    pam_unix.so try_first_pass
 auth     [default=die] pam_faillock.so authfail deny=5 even_deny_root audit unlock_time=900 root_unlock_time=60
